templates/masterclasses.html

```diff
@@ -52,6 +52,14 @@ <h3 class="p-media-object__title">
                   <li class="p-media-object__meta-list-item--date">
                     <span class="u-off-screen">Date: </span>{{ session.Date.Formatted }}
                   </li>
+                  <!-- Add to calendar button -->
+                  <!-- Currently, all events start at 11AM CST with duration of 45 minutes -->
```

Comment:
There is a duration column in the spreadsheet that we should probably use instead of assuming the 45-minute length.